assignment.js

```diff
@@ -112,6 +128,45 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+const playerAChoice = "scissors"
+const playerBChoice = "paper"
+
+if (playerAChoice === "rock"){
```

Comment:
@KatieGarrison15 nice work implementing logic that covers all of the possible outcomes. 

The approach that you're using involves nested `if` statements and while it works, you will want to avoid nesting if possible. The reason is that nesting can add complexity to your code which makes things more challenging to debug. However, your code is still easy to follow.